### PR TITLE
Ready View for making sure bamboohr Creds are valid

### PR DESCRIPTION
**Pull request by Ashutosh619-sudo**

### CODE REVIEW

File: apps/bamboohr/views.py [hunk 1]
```diff
@@ -12,9 +13,38 @@
...
+
 logger = logging.getLogger(__name__)
 logger.level = logging.INFO
 
+class BambooHrReadyView(generics.ListAPIView):
+
+    def get(self, request, *args, **kwargs):
+        try:
+            # bamboohr = BambooHr.objects.get(org_id__in=[kwargs['org_id']])
+            bamboohrsdk = BambooHrSDK(api_token='864acbfb14b38f974439af49ec95bc02e52a45b9', sub_domain='baba')
```
Comment (NileshPant1999) [inline apps/bamboohr/views.py:26]: please fix this @Ashutosh619-sudo

File: apps/bamboohr/views.py [hunk 2]
```diff
@@ -12,9 +13,38 @@
...
+                        'message': 'Ready'
+                    },
+                    status=status.HTTP_200_OK
+                )
+            else:
+                return Response(
+                    data = {
+                        'message': 'Invalid token'
+                    },
+                    status=status.HTTP_400_BAD_REQUEST
```
Comment (NileshPant1999) [inline apps/bamboohr/views.py:41]: 403 will be good here 
cc @ashwin1111

File: apps/bamboohr/views.py (hunk 1 above)
Comment (Ashutosh619-sudo) [reply apps/bamboohr/views.py:26]: are

File: apps/bamboohr/views.py [hunk 3]
```diff
@@ -12,9 +13,40 @@
 from apps.bamboohr.actions import disconnect_bamboohr, sync_employees
 from apps.names import BAMBOO_HR
 
+from rest_framework.views import APIView
+
 logger = logging.getLogger(__name__)
 logger.level = logging.INFO
 
+class BambooHrReadyView(generics.ListAPIView):
```
Comment (ashwin1111) [inline apps/bamboohr/views.py:21]: change class name similar to url

**Review by ashwin1111 — APPROVED** [review]

**Review by NileshPant1999 — APPROVED** [review]

**Pull request merged**